README.md

```diff
@@ -54,76 +54,75 @@ The project is primarily composed of two areas:
+See [bug raised with the library](https://github.com/hey-red/Mime/issues/36) for more info.
 
-**Mac only**
+Fix `magic.mgc` file version mismatches:
 
-   Install and link libmagic:
+If you're using Ubuntu 22.04, or later you'll most likely be using a `libmagic-1` binary that isn't 
+synchronized correctly with the `magic.mgc` file distributed with the build. This results in the 
+processor service failing during validation.
 
-    ```sh
-    brew install libmagic
-    brew link libmagic
-    env ARCHFLAGS="-arch x86_64" sudo gem install ruby-filemagic -- --with-magic-include=/usr/local/include --with-magic-lib=/usr/local/lib/
-    ```
+To fix this, you need to create an `appsettings.Local.json` file in the relevant project and ensure
+it has the following:
 
+```json
+{
+  "MagicFilePath": "/usr/lib/file/magic.mgc"
+}
+```
 
-  Download and link the `dylib` magic file.
+The above should work for Ubuntu, but you can change this path to whatever you want on your filesystem.
 
-    Depending on the arch of your machine, you will need to download a different dylib file dependent on the arch of your machine. You can find the correct file [in the runtimes section](https://github.com/hey-red/Mime/tree/master/src/Mime/runtimes) of the library
+#### Mac only
 
-    Download this file and place it somewhere where you won't accidentally delete it. Then link it to the correct location:
+Install and link `libmagic`:
 
-    ```sh
-    sudo ln -s /Users/${whoami}/path/to/folder/libmagic-1.dylib /usr/local/liblibmagic-1
-    sudo ln -s /Users/${whoami}/path/to/folder/libmagic-1.dylib /usr/local/lib/liblibmagic-1
-    ```
+```sh
+brew install libmagic
+brew link libmagic
+env ARCHFLAGS="-arch x86_64" sudo gem install ruby-filemagic -- --with-magic-include=/usr/local/include --with-magic-lib=/usr/local/lib/
+```
 
-  Upgrade version of `Mime` in `src/GovUk.Education.ExploreEducationStatistics.Common`
+Download and link the `dylib` magic file.
 
-  You'll need to upgrade the version of Mime to `3.4.0` in order for it to work with the new version of `libmagic`. You can do this by editing the `Mime` dependency in the `src/GovUk.Education.ExploreEducationStatistics.Common/` cs proj file:
+Depending on the arch of your machine, you will need to download a different dylib file dependent on the arch of your machine. You can find the correct file [in the runtimes section](https://github.com/hey-red/Mime/tree/master/src/Mime/runtimes) of the library
 
-  ```sh
-    <PackageReference Include="Mime" Version="3.4.0" />
-  ```
+Download this file and place it somewhere where you won't accidentally delete it. Then link it to the correct location:
 
-  you'll then need to rebuild the project to ensure changes take effect.
+```sh
+sudo ln -s /Users/${whoami}/path/to/folder/libmagic-1.dylib /usr/local/liblibmagic-1
+sudo ln -s /Users/${whoami}/path/to/folder/libmagic-1.dylib /usr/local/lib/liblibmagic-1
+```
```

Comment:
I'm reticent to outright recommend this as Mac advice as we literally have no one using a Mac on the project anymore.

However, I've slightly adjusted the README so that it potentially offers this a problem/solution on Mac as well (but needs confirmation).

Reply:
OK that sounds very sensible to me :D